### ISSUE TITLE
What is the difference of i3d and 3d ConvNet?

### ISSUE THREAD

**Issue by cemilzalluhoglu** [deepmind/kinetics-i3d#64]

**Comment by joaoluiscarreira** [comment]
Hi,

I3D is a 3D ConvNet.

Joao

**Comment by cemilzalluhoglu** [comment]
Then why are there two different lines with different names in Table 2 on the paper?
3D-ConvNet and Two Stream I3D. It can be written Two Stream 3D ConvNet. Am I right?

**Comment by joaoluiscarreira** [comment]
Yes, but they are two different 3d convnets. We could have written C3D and
Two Stream I3D.

**Comment by cemilzalluhoglu** [comment]
Is there a difference between C3D and I3D?

**Comment by joaoluiscarreira** [comment]
Yes, they have different structures.

**Comment by cemilzalluhoglu** [comment]
Could you explain the differences?

**Comment by joaoluiscarreira** [comment]
I3D is deeper, has fewer parameters, and has a multi-branch structure.

Joao

**Comment by cemilzalluhoglu** [comment]
I understand you. Can you explain the differences with examples?

**Comment by niderhoff** [comment]
@cemilzalluhoglu [Here](http://blog.qure.ai/notes/deep-learning-for-videos-action-recognition-review) is a really good article with examples.
